back/app/models/group.rb

```diff
@@ -85,7 +85,10 @@ def self.membership_types
   private
 
   def generate_slug
-    self.slug ||= SlugService.new.generate_slug self, title_multiloc.values.first
+    self.slug ||= SlugService.new.generate_slug(
+      self,
+      title_multiloc.find { |_, text| text.present? }&.last
```

Comment:
Just a small suggestion to make it more obvious that these are hash values
```suggestion
      title_multiloc.find { |_key, value| value.present? }&.last
```